app/models/mixins/ui_service_mixin.rb

```diff
@@ -1,21 +1,24 @@
 module UiServiceMixin
+  # rubocop:disable Metrics/LineLength
```

Comment:
yes I agree it's not ideal, however mixing source code and external validations that are ran on it is also not a good practice. today it's rubocop, tomorrow it's something else, putting those comments in code will force doing a major cleanup if rubocop is replaced. if there are some things and rules that need to be fine tuned in rubocop, that can and should be discussed.